book/src/install.md

```diff
@@ -156,6 +156,10 @@ pacman -S mingw-w64-ucrt-x86_64-helix
 
 Requirements:
 
+Clone the Helix GitHub repository into a directory of your choice. The
+examples in this documentation assume installation into either `~/src/` on
+Linux and macOS, or `%userprofile%\src\helix\runtime` on Windows.
```

Comment:
To match with the Linux instructions I think we should drop the ``helix\runtime`` and have it just be `%userprofile%\src\`

Reply:
Good call @gabydd